stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -281,6 +289,14 @@ public sealed interface ChatDomain {
      * @see <a href="https://getstream.io/chat/docs/query_channels/?language=kotlin">Filter syntax</a>
      */
     @CheckResult
+    @Deprecated(
+        message = "queryChannelsLoadMore is deprecated. Use extension function ChatClient::queryChannels instead",
```

Comment:
I believe we can't deprecate until we release `OfflinePlugin`. Right now there is no other public mechanism of querying more